aea/decision_maker/gop.py

```diff
@@ -45,8 +45,6 @@
 UtilityParams = Dict[str, float]  # a map from identifier to quantity
 ExchangeParams = Dict[str, float]  # a map from identifier to quantity
 
-QUANTITY_SHIFT = 100
```

Comment:
No need for this parameter anymore. This is set internally in the function to ensure consistent usage.